pyproject.toml

```diff
@@ -4,9 +4,9 @@
 [project]
 name = "postgresql-charms-single-kernel"
 description = "Shared and reusable code for PostgreSQL-related charms"
-version = "16.1.2"
+version = "16.1.3"
 readme = "README.md"
-license = "Apache-2.0"
+license = {file = "LICENSE"}
```

Comment:
UV plugin complains due to the license not being a file: https://github.com/canonical/pgbouncer-operator/actions/runs/19366235067/job/55410021051?pr=590#step:11:404